js/admin/tinymce.inc.js

```diff
@@ -34,7 +35,7 @@ function tinySetup(config)
 		selector: ".rte" ,
 		plugins : "colorpicker link image filemanager table media placeholder",
 		browser_spellcheck : true,
-		toolbar1 : "colorpicker,bold,italic,underline,strikethrough,blockquote,link,alignleft,aligncenter,alignright,alignfull,bullist,numlist,image",
+		toolbar1 : "colorpicker,bold,italic,underline,strikethrough,blockquote,link,alignleft,aligncenter,alignright,alignjustify,bullist,numlist,image",
```

Comment:
i don't see `alignfull` in documentation 

https://www.tinymce.com/docs/advanced/editor-control-identifiers/#toolbarcontrols